enums.h

```diff
@@ -1310,7 +1310,7 @@ typedef enum missile_graphic_id {
 	MFILE_SCBSEXPD,
 	MFILE_NULL,
 	MFILE_INVALID,

```

Comment:
Also, I think this enum entry can be removed. `MFILE_INVALID` is never used and does not correspond to a valid array index of missiledata.

Reply:
```suggestion
```